pkg/receive/handler.go

```diff
@@ -275,6 +277,18 @@ func NewHandler(logger log.Logger, o *Options) *Handler {
 		),
 	)
 
+	h.router.Post(
+		"/v1/metrics",
```

Comment:
Maybe endpoint should be something like this, to be inline with rest of the APIs?
```suggestion
		"/api/v1/otlp",
```